docs/source/configuration.rst

```diff
@@ -699,6 +699,16 @@ ignored until a corresponding `-- noqa:enable=<rule>[,...] | all` directive.
 
 .. _sqlfluffignore:
 
+Ignoring types of errors
+^^^^^^^^^^^^^^^^^^^^^^^^
+General *categories* of errors can be ignored using the ``--ignore`` command
+line option or the ``ignore`` setting in ``.sqlfluffignore``. Types of errors
+that can be ignored include:
+
+* ``lexing``
+* ``parsing``
+* ``templating``
```

Comment:
You can actually also ignore all `linting` errors too!

```suggestion
* ``lexing``
* ``linting``
* ``parsing``
* ``templating``
```

Reply:
Yes. I was wondering if we wanted to advertise that. I'll make the change. Are you aware of users wanting to do this?

Reply:
Nope! But not aware of anyone regularly wanting to do the others either (and wouldn't advise it! It suggests something more fundamental is wrong).

Of all of those, ignoring linting errors is the LEAST likely to cause real problems IMHO 😁 Though, at the same time, it also negates much of the benefit of using SQLFluff!